.github/workflows/test_on_push.yaml

```diff
@@ -20,8 +20,12 @@ jobs:
         with:
           tarantool-version: '${{ matrix.tarantool }}'
 
+      - name: Add the repository for tt utility
```

Comment:
Please name this step `Install tt utility` and move `sudo apt-get -y install tt` to this step.